JetNews/app/src/main/java/com/example/jetnews/ui/Status.kt

```diff
@@ -28,9 +29,8 @@ sealed class Screen {
     object Interests : Screen()
 }
 
-@Model
 object JetnewsStatus {
-    var currentScreen: Screen = Screen.Home
+    var currentScreen: MutableState<Screen> = mutableStateOf(Screen.Home)
```

Comment:
Can we just expose the `Screen` and use `state` as an implementation detail here.

Reply:
Used delegate